templates/data/opensearch/index.html

```diff
@@ -0,0 +1,336 @@
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="u-no-margin--bottom" />
+    <div class="col-6 col-medium-3">
+      <h2>Fully automated <br class="u-hide--small" />application operations with Charmed OpenSearch</h2>
+    </div>
+    <div class="col-6 col-medium-3">
+      <p>
+        OpenSearch is a fantastic open source search and analytics suite. However, operating and supporting a production OpenSearch solution can be complex and challenging.
+      </p>
+      <p>
+        Canonical and the community are developing an open source operator called  Charmed OpenSearch, making it easier to operate OpenSearch on physical,  Virtual Machines (VM) and other wide range of cloud and cloud-like environments in an automated way. 
+      </p>
+      <p>
+        The Charmed OpenSearch operator is available for the open source community to use. Learn more about the ‘edge’ version of the operator in the  <a href="https://charmhub.io/opensearch">Charmed OpenSearch Technical documentation</a>.
+      </p>
+      <p>
+        <a href="/data/opensearch#get-in-touch" class="js-invoke-modal">
+          Notify me when the operator enters General Availability&nbsp;&rsaquo;
+        </a>
+      </p>
+      <div class="p-strip--light u-no-padding--top u-no-padding--bottom u-align--center u-vertically-center u-aspect-ratio--3-2">
+        {{ 
+          image (
+            url="https://assets.ubuntu.com/v1/a8147e27-charmed-opensearch-application-operations.svg",
+            alt="", 
+            width="230", 
+            height="148", 
+            hi_def=True, 
+            loading="lazy" 
+          ) | safe
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip u-no-padding--top">
+  <div class="row">
+    <hr class="u-no-margin--bottom" />
+    <div class="col-6 col-medium-3">
+      <h2>Secure and scale your <br class="u-hide--small" />open-source data applications</h2>
+    </div>
+    <div
+      class="col-6 col-medium-3"
+    >
+      <p>
+        Unlock the potential of your data with an integrated, open source platform for applications and infrastructure. Canonical also offers enterprise solutions for data applications such as <a href="/data/mongodb">MongoDB</a>, <a href="/data/spark">Spark</a> and <a href="/data/kafka">Kafka</a>.
+      </p>
+      <p>
+        <a
+          href="/data/opensearch#get-in-touch"
+          class="p-button--positive js-invoke-modal"
+          data-testid="interactive-form-link"
+        >
+          Contact us
+        </a>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section>
+  <div class="row">
+    <hr class="u-no-margin--bottom" />
+    <div class="col-3 col-medium-3">
+      <h2 class="p-heading--2">Resources</h2>
```

Comment:
This heading isn't on the copy doc

Reply:
The heading was added in the design, and Michelle went through it and didn't seem to have a problem with it, so I don't know, should I add it to the copy doc? How does this work?

Reply:
The doc is the source of truth for copy always, so I'd reach out to the pdm and ask her to update the doc or you can also update it yourself if she's confirmed that she's ok with that change

Reply:
I've spoken to her, she okayed the changes to the copy doc.